<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFunctionDialog</class>
 <widget class="QDialog" name="EditFunctionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Function</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name of function</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="startLabel">
       <property name="text">
        <string>Start address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="startLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="stackSizeLabel">
       <property name="text">
        <string>Stack size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="stackSizeLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="callConLabel">
       <property name="text">
        <string>Calling convention</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="callConComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditFunctionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFunctionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
